personhood/contract_spawner.go

```diff
@@ -111,8 +111,9 @@ func (c *ContractSpawner) Spawn(rst byzcoin.ReadOnlyStateTrie, inst byzcoin.Inst
 			"coin\\.(fetch|store|transfer)|" +
 			"credential\\.(update|recover)|" +
 			"popParty\\.(barrier|finalize|mine|addParty)|" +
-			"ropasci\\.(second|confirm)" +
-			"value\\.update))$")
+			"ropasci\\.(second|confirm)|" +
```

Comment:
> When using the spawner-instance, darcs with spawn:calypsoRead should
be allowed, as they can only be used as instructions to a calypsoWrite
instance that will check if coins are needed or not.

This explanation should be added in the comment above.